utils/rule_dir_json.py

```diff
@@ -50,6 +50,7 @@ def walk_products(root, all_products):
         product_dir = os.path.join(root, "products", product)
         product_yaml_path = os.path.join(product_dir, "product.yml")
         product_yaml = ssg.products.load_product_yaml(product_yaml_path)
+        product_yaml.read_properties_from_directory(os.path.join(root, "product_properties"))
```

Comment:
Would it make sense to use  the product.yml (from the `build` directory) instead of reading the source product.yml and product_properties?

Reply:
The rule dir json operates without build artifacts.